README.md

```diff
@@ -294,6 +294,41 @@ npm run lint
 npm run format
 ```
 
+## 👩‍🎤 Storybook
+
+Storybook helps us build UI components in isolation from our app's business logic, data, and context. That makes it easy to develop hard-to-reach states and save these UI states as stories to revisit during development, testing, or QA.
+
+To start adding stories, create a `index.stories.tsx` inside the component's folder:
+
+<pre>
+src
+└─── components
+│   └─── @shared
+│       └─── <your component>
+│            │   index.tsx
+│            │   index.module.css
+│            │   <b>index.stories.tsx</b>
+│            │   index.test.tsx
+</pre>
+
+You can also write a [test](https://storybook.js.org/docs/react/writing-tests/importing-stories-in-tests#example-with-testing-library) against your story by creating a `index.test.tsx` file.
```

Comment:
ideally, every story we write is automatically run as a unit test, without having to write a story and a test file. So the use case is that every component should have a story and that in most cases covers unit testing the component. Only when we have more advanced test use cases we should have to write a dedidcated test file on top of the story file.

Reply:
Maybe I'm misunderstanding the [documentation](https://storybook.js.org/docs/react/writing-tests/importing-stories-in-tests), but it seems like you necessarily need a test file to cover unit testing. I don't see how we could automatically run these stories as unit tests without having a test file as it seems required.

Reply:
there was some way but not sure if still feasible. Did some archeology and this is how we did that 2 years ago:
https://github.com/oceanprotocol/market/blob/v0.2.0/tests/unit/storyshots.test.tsx

So we used [addon-storyshots](https://storybook.js.org/addons/@storybook/addon-storyshots) and made it run through react-testing-library as far as I understand the code comment

Reply:
It's still possible. This is actually pretty awesome 🤯. Now, it's been setup 👍🏽